build/charts/antrea/crds/externalnode.yaml

```diff
@@ -11,12 +11,18 @@ spec:
       schema:
         openAPIV3Schema:
           type: object
+          required:
+            - spec
           properties:
             spec:
               type: object
+              required:
+                - interfaces
               properties:
                 interfaces:
                   type: array
+                  required:
+                    - ips
```

Comment:
Does it require at least one interface and one IP? if it just sets "required", this input will pass and the interfaces and ips are still empty:
```
- spec:
    interfaces: []
---
- spec:
    interfaces:
    - ips: []
```

Reply:
Thanks a lot. It is true that these empty array will pass the validation. I further add minItems for interfaces and ips, please help review the latest version.